package.json

```diff
@@ -5,9 +5,11 @@
   "homepage": "http://softlayer.github.io/sl-ember-components",
   "scripts": {
     "build": "ember build",
+    "docs": "ember ember-cli-jsdoc",
+    "lint": "node_modules/.bin/sl-eslint",
+    "lint-all": "node_modules/.bin/sl-eslint addon app tests",
```

Comment:
Any folder we point JSDoc at we should also point ESLint at.  Per https://github.com/softlayer/sl-ember-components/blob/develop/jsdoc.json#L9 add the `blueprints` folder.